pkg/generators/models/models.go

```diff
@@ -580,6 +593,23 @@ func fillValidationRelatedProperties(ref *openapi3.SchemaRef, spec *PropSpec) (i
 		spec.HasFormat = true
 	}
 
+	if ref.Value.Pattern != "" {
+		spec.NeedsValidation = true
+		spec.Pattern = ref.Value.Pattern
+		importsMap["regexp"] = something{}
+		msg, ok := ref.Value.ExtensionProps.Extensions[ExtensionPatternError]
+		if ok {
+			rawMsg, ok := msg.(json.RawMessage)
```

Comment:
apparently the extensions are actually json RawMessage, but there is not clear documentation on this, even though it is a `map[string]interface{}`